GP-Engine-team/GP_Engine
# Fixed shared inputs on overlapping windows

## Description
The scene editor and the game view both react to user input depending on the user's actions.
Sometimes, these windows overlap, and the window under the top one react to inputs.
This pull request removes this behaviour.

## How to test
1. Detach the game viewer window or the scene editor window, and overlap them with one another. For instance:
![image](https://user-images.githubusercontent.com/33126700/120925859-11d81780-c6ca-11eb-819f-5683abd6cf29.png)
2. Play the game
3. If the game viewer is on top, check that inputs are not captured by the scene editor.
If the scene editor is on top, check that the game viewer is not capturing inputs in any way
